test/indexer/test_hca_indexer.py

```diff
@@ -994,9 +995,43 @@ def test_invalid_auth_for_notification_request(self):
         }
         for endpoint in ['/', '/delete']:
             with self.subTest(endpoint=endpoint):
-                response = self._test(body, endpoint='/', valid_auth=False)
+                response = self._test(body, endpoint=endpoint, valid_auth=False)
                 self.assertEqual(401, response.status_code)
 
+    @mock_sts
+    @mock_sqs
+    def test_index_test_mode(self):
+        self._create_mock_notify_queue()
+        testless_notification = {
+            "match": {
+                "bundle_uuid": "bb2365b9-5a5b-436f-92e3-4fc6d86a9efd",
+                "bundle_version": "2018-03-28T13:55:26.044Z"
+            }
+        }
+        test_name_in_notification = {
+            "match": {
+                "bundle_uuid": "bb2365b9-5a5b-436f-92e3-4fc6d86a9efd",
+                "bundle_version": "2018-03-28T13:55:26.044Z"
+            },
+            "test_name": "integration-test_bb2365b9-5a5b-436f-92e3-4fc6d86a9efd_bb2365b9"
+        }
+        for endpoint in ['/', '/delete']:
+            for test_mode in ('0', '1'):
```

Comment:
```suggestion
            for test_mode in 0, 1:
```

Tuple parentheses are optional after `in` in `for` statements.

And I would convert to `str` when patching os.environ